arc/cli/utils/console.py

```
@@ -3,25 +3,32 @@
 from rich.console import Console
 from rich.theme import Theme
 from rich.text import Text
-from typing import Any
+from rich.live import Live
+from rich.progress import Progress
```

Ruff (0.11.9)

`rich.progress.Progress` imported but unused

Remove unused import: `rich.progress.Progress`

(F401)